heroic-core/src/main/java/com/spotify/heroic/cluster/TracingClusterNode.java

```diff
@@ -68,6 +70,13 @@ public NodeMetadata metadata() {
         return new Group(delegateNode.useOptionalGroup(group));
     }
 
+    public <T> Transform<Throwable, T> wrapExceptionTransform() {
+        return throwable -> {
+            // Actually never return;s, instead throws a new exception with added info
+            throw new RuntimeNodeException(queryIdentifier.toString(), throwable.getMessage());
```

Comment:
Discards the original throwable, it should be set as cause - otherwise we lose vital troubleshooting information in stack traces.